src/routes/cache-hooks/entities/safe-apps-update.entity.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  EventPayload,
+  EventType,
+} from '@/routes/cache-hooks/entities/event-payload.entity';
+
+export interface SafeAppsUpdate
+  extends Omit<EventPayload<EventType.SAFE_APPS_UPDATE>, 'address'> {}
```

Comment:
Nit: no need to address this on the PR, but I think it could be a bit clearer if we strip off `address` from the `EventPayload` ancestor type instead of using `Omit` here. This would give us the opportunity to stay in-line with the AJV schema also (`webHookSchema`, which doesn't require an `address` anymore).

Reply:
Perfect, makes sense 👍